changelog.txt

```diff
@@ -2,6 +2,7 @@
 
 = 5.2.0 - 2022-xx-xx =
 * Add - New wcs_set_order_address() helper function to set an array of address fields on an order or subscription.
+* Add - Highlight subscriptions with overdue payment in list view with red icon & tooltip.
```

Comment:
This changelog entry should probably be moved to `5.3.0` since [`5.2.0` has already been tagged](https://github.com/Automattic/woocommerce-subscriptions-core/releases/tag/5.2.0).

Reply:
Good spotting - thanks @Jinksi ! Would this get caught & fixed during subs-core tag/"release" process?

Reply:
> Would this get caught & fixed during subs-core tag/"release" process?

Nah it wouldn't automatically be apparent that this changelog entry was added after 5.2.0 was tagged. :( 

I can just submit a small PR to move it up.

Reply:
Thanks James!

> Nah it wouldn't automatically be apparent that this changelog entry was added after 5.2.0 was tagged. :(

Dang, that's a tiny bit fragile. I think I personally look at a diff of the changelog when I did (a while ago?!) releases, which might catch it. IMO a good release process has one person eyeball the full changelog before it goes out, to catch duplicates, missing stuff, inconsistent wording etc.

Reply:
A diff like this is a good way to eyeball changelog changes before tagging: https://github.com/Automattic/woocommerce-subscriptions-core/compare/5.2.0...trunk#diff-b40cd67182487ef24807d9c9268329d35fbd96aa2b0a9cae69e2e0d746b1c666

Reply:
> Dang, that's a tiny bit fragile.

Yeah. 

Doing a mini-retro personally, I was thinking about how this happened. I put it down to the changelog in this PR's changeset shows the `5.2.0 22-xx-xx` which indicates it hasn't been tagged. The fact that it had indicates this branch would have been behind `trunk` and so I must have missed the "needs update" flag on this PR as well. 

So there are a few things that occurred that would have caught this. Updating the branch before merging and paying more attention to the version. 

> I think I personally look at a diff of the changelog when I did (a while ago?!) releases, which might catch it. IMO a good release process has one person eyeball the full changelog before it goes out, to catch duplicates, missing stuff, inconsistent wording etc.

This probably isn't the right place for this discussion to occur but I'd prefer we moved towards using Jetpack changelogs over introducing a manual process for releases which has fragility issues of its own.